packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -85,11 +85,8 @@ contract EpochRewards is
 
   event TargetVotingYieldUpdated(uint256 fraction);
 
-  modifier onlyVmOrPermitted(address permittedAddress) {
-    if (isL2()) require(msg.sender == permittedAddress, "Only permitted address can call");
-    else {
-      require(msg.sender == address(0), "Only VM can call");
-    }
+  modifier onlyPermitted(address permittedAddress) {
```

Comment:
This modifier is the same we have in the election contract: https://github.com/celo-org/celo-monorepo/pull/11353/files#diff-0244e44347e8596d09c833006475db2bea054c82b301ede72c7195213ce7e695R164

Would take the opportunity to abstract it away